File: app/src/main/java/org/docheinstein/minimote/settings/SettingsFragment.java
package org.docheinstein.minimote.settings;

import android.os.Bundle;
import android.text.InputType;

import org.docheinstein.minimote.R;
import org.docheinstein.minimote.base.MinimotePreferenceFragment;

public class SettingsFragment extends MinimotePreferenceFragment {
    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        setPreferencesFromResource(R.xml.preferences, rootKey);
    }
}

File: app/src/main/java/org/docheinstein/minimotek/ui/settings/SettingsFragment.kt
package org.docheinstein.minimotek.ui.settings

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import org.docheinstein.minimotek.R

class SettingsFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.preferences, rootKey)
    }
}